scripts/exregional_process_bufr.sh

```diff
@@ -0,0 +1,415 @@
+else
+  print_err_msg_exit "\
+The executable specified in EXEC does not exist:
+  EXEC = \"$EXEC\"
+Build lightning process and rerun."
+fi
+#
+#
+#-----------------------------------------------------------------------
+#
+# Run the process for lightning bufr file 
+#
+#-----------------------------------------------------------------------
+#
+if [ $run_lightning ]; then
+   $APRUN ./process_Lightning_bufr.exe > stdout_lightning_bufr 2>&1 || print_err_msg "\
+   Call to executable to run lightning process returned with nonzero exit code."
+fi
+
+#
+#-----------------------------------------------------------------------
+#-----------------------------------------------------------------------
+#
+# Link to the observation NASA LaRC cloud bufr file
+#
+#-----------------------------------------------------------------------
+
+obs_file=${OBSPATH}/${YYYYMMDDHH}.rap.t${HH}z.lgycld.tm00.bufr_d
+print_info_msg "$VERBOSE" "obsfile is $obs_file"
+run_cloud=false
+if [ -r "${obs_file}" ]; then
+   cp_vrfy "${obs_file}" "NASA_LaRC_cloud.bufr"
+   run_cloud=true
+else
+   print_info_msg "$VERBOSE" "Warning: ${obs_file} does not exist!"
+fi
+
+#-----------------------------------------------------------------------
+#
+# Build namelist and run executable for NASA LaRC cloud
+#
+#   analysis_time : process obs used for this analysis date (YYYYMMDDHH)
+#   bufrfile      : result BUFR file name
+#   npts_rad      : number of grid point to build search box (integer)
+#   ioption       : interpolation options
+#                   = 1 is nearest neighrhood
+#                   = 2 is median of cloudy fov
+#   bkversion     : grid type (background will be used in the analysis)
+#                   = 0 for ARW  (default)
+#                   = 1 for FV3LAM
+#-----------------------------------------------------------------------
+
+cat << EOF > namelist_nasalarc
+ &setup
+  analysis_time = ${YYYYMMDDHH},
+  bufrfile='NASALaRCCloudInGSI_bufr.bufr',
+  npts_rad=1,
+  ioption = 2,
+  bkversion=1,
+ /
+EOF
+
+#
+#-----------------------------------------------------------------------
+#
+# Copy the executable to the run directory.
+#
+#-----------------------------------------------------------------------
+#
+EXEC="${EXECDIR}/process_larccld.exe"
+
+if [ -f $EXEC ]; then
+  print_info_msg "$VERBOSE" "
+Copying the NASA LaRC cloud process  executable to the run directory..."
+  cp_vrfy ${EXEC} ${WORKDIR}/process_larccld.exe
+else
+  print_err_msg_exit "\
+The executable specified in EXEC does not exist:
+  EXEC = \"$EXEC\"
+Build lightning process and rerun."
+fi
+#
+#
+#-----------------------------------------------------------------------
+#
+# Run the process for NASA LaRc cloud  bufr file 
+#
+#-----------------------------------------------------------------------
+#
+if [ $run_cloud ]; then
+  $APRUN ./process_larccld.exe > stdout_nasalarc 2>&1 || print_err_msg "\
+  Call to executable to run NASA LaRC Cloud process returned with nonzero exit code."
+fi
```

Comment:
Please reduce the duplication of information here, similarly.

Reply:
fixed